api/SpeechRecognitionErrorEvent.json

```diff
@@ -1,21 +1,33 @@
 {
   "api": {
-    "SpeechRecognitionError": {
+    "SpeechRecognitionErrorEvent": {
       "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechRecognitionError",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechRecognitionErrorEvent",
         "support": {
-          "chrome": {
-            "prefix": "webkit",
-            "version_added": "33",
-            "notes": "You'll need to serve your code through a web server for recognition to work."
-          },
-          "chrome_android": {
-            "prefix": "webkit",
-            "version_added": true,
-            "notes": "You'll need to serve your code through a web server for recognition to work."
-          },
+          "chrome": [
+            {
+              "version_added": "77",
+              "notes": "You'll need to serve your code through a web server for recognition to work."
+            },
+            {
+              "alternative_name": "webkitSpeechRecognitionError",
+              "version_added": "33",
+              "version_removed": "77"
+            }
+          ],
+          "chrome_android": [
+            {
+              "version_added": "77",
+              "notes": "You'll need to serve your code through a web server for recognition to work."
+            },
+            {
+              "alternative_name": "webkitSpeechRecognitionError",
+              "version_added": "33",
+              "version_removed": "77"
+            }
+          ],
           "edge": {
-            "version_added": null
+            "version_added": false
```

Comment:
This should be “79”, along with all other Edge data.

Reply:
OK — changed